HISTORY

```diff
@@ -1,3 +1,6 @@
+=== 3.1.0 2018-06-04
+- Adds support for sessions decisions to [Decisions API](https://siftscience.com/developers/docs/curl/decisions-api)
```

Comment:
Maybe "get session decisions" is better wording. Depends are we returning all the decisions associated with a session or just the latest ?
Also do we have support for batch get sessions or just get session ?

Reply:
It's just get session, not batched